### DISCUSSION TITLE
Splitting the code examples into modules

### DISCUSSION THREAD

**Discussion by alexmozaidze** [bwmarrin/discordgo#1421] [Ideas]
I am trying to reason about slash commands and how to handle them with this library, but it was quite difficult to do because of how `slash_commands/main.go` was stuffed up by the irrelevant for me things. I understand that it attempts to show all the capabilities of the library, but I need a small portion of its functionality. Could you, perhaps, split this example into sub-examples?

I very much like the library, keep up the good work!

**Comment by KlartNET** [comment]
+1 i really want this.
